### PR TITLE
Cover with tests all new IronWorker features (label,tasks on pause)

### PR DESCRIPTION
**Pull request by sunloverz**

### CODE REVIEW

**Comment by sunloverz** [comment]
@thousandsofthem could you please review?

File: test/test_basic.rb [hunk 1]
```diff
@@ -20,4 +15,44 @@ def test_basic
...
+
+  def test_schedule_label
+    create_code('test_label')
+    resp = client.schedules.create('test_label', {}, start_at: Time.now + 120, run_times: 1, label: 'schedule_label' )
+    task = client.schedules.get(resp.id)
+    assert_equal "schedule_label", task.label
+  end
+
+  def test_pause_task_queue
+    code_id = create_code('test_paused_task')
```
Comment (rkononov) [inline test/test_basic.rb]: could you also test that new tasks are not running after pausing code

File: test/test_basic.rb [hunk 2]
```diff
@@ -20,4 +15,44 @@ def test_basic
...
+    client.tasks.create('test_paused_task')
+    response = client.codes.pause_task_queue(code_id)
+    paused_code = client.codes.get(code_id)
+    assert_equal response['msg'], 'Paused'
+    assert_equal paused_code.max_concurrency, -1
+    client.codes.resume_task_queue(code_id)
+  end
+
+  def test_resume_task_queue
```
Comment (rkononov) [inline test/test_basic.rb]: and new tasks are running after resume

**Comment by sunloverz** [comment]
Roman done!

**Pull request merged**